### DISCUSSION TITLE
Ui differences

### DISCUSSION THREAD

**Discussion by Angelk90** [TeamNewPipe/NewPipe#10701] [Questions]
These are some screenshots of the ui between the various tabs:

| Yt  | NewPipe Tablet Grid mode |
| ------------- | ------------- |
| ![1](https://github.com/TeamNewPipe/NewPipe/assets/20476002/3c3fcef6-8aa3-4cc7-ae86-9aaff8ebd894) |  |
| ![2](https://github.com/TeamNewPipe/NewPipe/assets/20476002/2d06f32d-4ec3-426e-9dd1-938f99670290) | ![2-](https://github.com/TeamNewPipe/NewPipe/assets/20476002/a230ba27-2e5d-48a3-96f5-9b38853d9974) |
| ![3](https://github.com/TeamNewPipe/NewPipe/assets/20476002/2f55fd35-3861-44e2-a1a8-0314b09c81b4) | ![3-](https://github.com/TeamNewPipe/NewPipe/assets/20476002/f9c57aee-e66c-4eba-b09e-3bddc9e21461) |
| ![4](https://github.com/TeamNewPipe/NewPipe/assets/20476002/d4eb4119-51cc-4219-a6a4-ebdc76f3e5bf) | ![4-](https://github.com/TeamNewPipe/NewPipe/assets/20476002/dcc511c2-9bdb-4f48-a7a1-97594ddbbbb3) |
| ![5](https://github.com/TeamNewPipe/NewPipe/assets/20476002/390662c6-30c6-41f5-b158-5029802df6f9) |   |
| ![6](https://github.com/TeamNewPipe/NewPipe/assets/20476002/bd4cb7f4-5132-4ff4-a4d1-94080717ad74) | ![6-](https://github.com/TeamNewPipe/NewPipe/assets/20476002/a2b997c3-07a5-4c3e-a171-4ed1176087f6) |
| ![7](https://github.com/TeamNewPipe/NewPipe/assets/20476002/1e363c8a-ad54-42c9-a7ad-6304aea2431b) |  |
|  | ![8-](https://github.com/TeamNewPipe/NewPipe/assets/20476002/f0f5d999-84cd-4a36-b9fe-968d48455d9d) |

1) In screenshot 1 on Yt, see black rectangle, information such as:
- Channel username: `@cs50`
- How many videos have been uploaded to the channel
- Various social networks to which the channel is connected

2) In the various screenshots on NewPipe, seeing the black rectangle shows which channel the video, short, live and playlist refers to.
A bit useless in this case, since it is already on the cs50 channel, it makes no sense to show this information there.
It makes sense to show it when we are on trends, history, etc., in a part that is not the channel.

3) The podcast and community part is missing on NewPipe.

4) Why do we use all capital letters, like tabs and buttons.

5) The video previews, in videos, shorts, live in grid mode there is too much space between one preview and another, so you have too small preview images.

**Comment by Angelk90** [comment]
@TeamNewPipe , @TobiGr, @Stypox, @AudricV : 
I would like to discuss these UI differences with you, with the entire NewPipe team.
On what the future of NewPipe might be.

**Reply by opusforlife2** [reply]
Please avoid mass pings. Just commenting without pinging is also enough to draw our attention.

If you're interested in real-time conversation then come to the Matrix room from the ReadMe.

**Comment by Stypox** [comment]
I guess it makes sense to remove the useless channel names. This should be quite easy to do with Compose on the `refactor` branch.

**Reply by Angelk90** [reply]
What exactly do you mean?
